### PR TITLE
WIP: remove and clean code

### PR DESCRIPTION
**Pull request by felipeZ**
Remove pymonad #156  
TODO:
* [x] remove `zipwith`
* [x] remove `zipwith3`
* [x] remove curry decorator

### CODE REVIEW

File: test/test_using_plams/test_opt_orca.py [hunk 1]
```diff
@@ -67,7 +64,8 @@ def test_methanol_opt_orca():
 
     coords = collapse([a.coords for a in mol_opt.atoms])
 
-    assert abs(sum(zipWith(operator.sub)(coords)(expected_coords))) < 1e-7
+    assert abs(sum(real - expected for (real, expected)
```
Comment (BvB93) [inline test/test_using_plams/test_opt_orca.py]: You could consider using [``math.isclose()``](https://docs.python.org/3/library/math.html#math.isclose) here. Note that the ``1e-7`` would be equivalent here to the ``abs_tol`` argument, not ``rel_tol``.

File: test/test_using_plams/test_opt_gamess.py [hunk 2]
```diff
@@ -42,7 +41,8 @@ def test_opt_gamess():
                        1.1448447102, 0.562978981, 0.9027182521,
                        1.1454516521, -0.9993402516, 1.04943e-05]
 
-    assert abs(sum(zipWith(operator.sub)(coords)(expected_coords))) < 1e-7
+    assert abs(sum(real - expected for (real, expected)
```
Comment (BvB93) [inline test/test_using_plams/test_opt_gamess.py]: See https://github.com/SCM-NV/qmflows/pull/157/files#r392198476.

File: test/test_cp2k_parser.py [hunk 3]
```diff
@@ -1,12 +1,13 @@
...
+from pathlib import Path
 from qmflows.parsers import cp2KParser
-import os
 
 
 def test_read_cp2k_coefficients():
     """Test that the CP2K output coefficients are read properly."""
-    path = "test/test_files/output_cp2k"
-    mo_file = os.path.join(path, "mo_coeffs.log")
+    path = Path("test/test_files/output_cp2k")
```
Comment (BvB93) [inline test/test_cp2k_parser.py]: I'd recommend changing this into ``path = Path('test') / 'test_files' / 'output_cp2k')``; should cause less issues on different operating systems.

**Comment by BvB93** [comment]
That's about it.

File: test/test_cp2k_parser.py (hunk 3 above)
Comment (BvB93) [reply test/test_cp2k_parser.py]: There is also the constant (located in ``qmflows.test_utils.PATH``) which already points to ``"test/test_files"``.

File: test/test_using_plams/test_opt_orca.py (hunk 1 above)
Comment (felipeZ) [reply test/test_using_plams/test_opt_orca.py]: :+1:

**Pull request merged**